events.md

```diff
@@ -7,4 +7,4 @@ navigation_order: 1
 
 These are the planned events for Monthly Music Hackathon. Check out each one for more details and to RSVP.
 
-For past events, check in the [Event Archive](/events/archive.html).
```

Comment:
My bad! What I meant was if you look in _site you'll see:

permalink: /events/archive produces
_site/events/archive.html

and 

permalink: /events/archive/
produces
_site/events/archive/index.html

Check out 
https://jekyllrb.com/docs/structure/
and 
https://jekyllrb.com/docs/permalinks/
for more detail!